docs/mkdocs.yml

```diff
@@ -72,6 +72,9 @@ pages:
     - Optimization Algorithms: APIGuide/Optimizers/Optim-Methods.md
   - Triggers: APIGuide/Triggers.md
   - Metrics:  APIGuide/Metrics.md
+  - Spark ML:
```

Comment:
I think we can merge the DLEstimator and DLClassifier files into one file

Reply:
Done.

Reply:
It's still two md files, no?